src/contracts/pods/EigenPod.sol

```diff
@@ -552,7 +548,13 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         }
     }
 
-    function _processPartialWithdrawal(uint64 withdrawalHappenedSlot, uint64 partialWithdrawalAmountGwei, uint40 validatorIndex, bytes32 validatorPubkeyHash, address recipient) internal {
+    function _processPartialWithdrawal(
```

Comment:
can we standardize the order of parameters `_processPartialWithdrawal` vs in `_processFullWithdrawal`?

Reply:
yes, will make a commit with this

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/135/commits/2c8b0c20b430307967a96fabf58c362fb9e400f0